src/domain/accounts/index.ts

```diff
@@ -22,11 +23,34 @@ export * from "./primitives"
 const UserIdRegex =
   /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i
 
+// device id format from AppCheck: 1:72279297366:android:35666807ae916c5aa75af7
+const DeviceIdRegex = /^[0-9]+:[0-9]+:[a-z]+:[0-9a-z]+$/i
+
+export const checkedToDeviceAccountUserId = (userId: string) => {
+  if (userId.match(/^[0-9]+:[0-9]+:[a-z]+:[0-9a-z]+$/i)) {
```

Comment:
```suggestion
  if (userId.match(DeviceIdRegex) {
```

is this what you meant here?